app/services/analytics_events.rb

```diff
@@ -4708,6 +4708,101 @@ def idv_socure_shadow_mode_proofing_result_missing(**extra)
     track_event(:idv_socure_shadow_mode_proofing_result_missing, **extra)
   end
 
+  # @param [Boolean] success Whether form validation was successful
+  # @param [Hash] errors Errors resulting from form validation
+  # @param [String] exception
+  # @param [Boolean] billed
+  # @param [String] docv_transaction_token
+  # @param [Hash] customer_profile
+  # @param [String] reference_id
+  # @param [Hash] reason_codes
+  # @param [Hash] document_type
+  # @param [Hash] decision
+  # @param [String] user_id
+  # @param [String] state
+  # @param [String] state_id_type
+  # @param [Boolean] async
+  # @param [Integer] submit_attempts Times that user has tried submitting (previously called
+  # "attempts")
+  # @param [Integer] remaining_submit_attempts (previously called "remaining_attempts")
+  # @param ["hybrid","standard"] flow_path Document capture user flow
+  # @param [Float] vendor_request_time_in_ms Time it took to upload images & get a response.
+  # @param [Boolean] doc_type_supported
+  # @param [Boolean] doc_auth_success
+  # @param [Boolean] liveness_checking_required Whether or not the selfie is required
+  # @param [Boolean] liveness_enabled Whether or not the selfie result is included in response
+  # @param [String] vendor
+  # @param [Boolean] address_line2_present
+  # @param [String] zip_code
+  # @param [String] birth_year Birth year from document
+  # @param [Integer] issue_year Year document was issued
+  # @param [Boolean] biometric_comparison_required
+  # The request for socure verification was sent
+  def idv_socure_verification_data_requested(
+    success:,
+    errors:,
+    exception:,
+    async:,
+    docv_transaction_token: nil,
```

Comment:
```suggestion
    async:,
    docv_transaction_token: nil,
    exception: nil,
```

adding compact when calling analytics ... exception may be empty if there is no exception